src/python/WMCore/BossAir/MySQL/RunJobByStatus.py

```diff
@@ -0,0 +1,45 @@
+"""
+_RunJobByStatus_
+
+Monitoring DAO classes for Jobs in BossAir database.
+It groups jobs in each sched_status and bossAir status and guarantee
+all sched_status are always present in the output.
+"""
+from __future__ import print_function, division
+
+from WMCore.Database.DBFormatter import DBFormatter
+
+
+class RunJobByStatus(DBFormatter):
+    sql = """
+          SELECT bl_status.name AS sched_status, count(bl_runjob.sched_status) AS count, bl_runjob.status
```

Comment:
Alan, does sched_status point to actual schedd states, IDLE, Removed, Held, etc or mapped states Pending, Running, etc?

Reply:
Yes, I figured it would to more good than harm.